cmd/dockerd/config_unix_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func TestDaemonParseShmSize(t *testing.T) {
 	if runtime.GOOS == "solaris" {
-		t.Skip("ShmSize not supported on Solaris\n")
+		t.Skip("ShmSize not supported on %s", runtime.GOOS)
```

Comment:
This removes the trailing `\n`, is that correct?

t.b.h., this is just a test, and `Solaris` to me is just fine as well